docs/articles/documentation/using-testcafe/programming-interface/runner.md

```diff
@@ -67,7 +67,11 @@ Parameter | Type                | Description
 --------- | ------------------- | ----------------------------------------------------------------------------
 `source`  | String &#124; Array | The relative or absolute path to a test fixture file, or several such paths. You can use [glob patterns](https://github.com/isaacs/node-glob#glob-primer) to include (or exclude) multiple files.
 
-If you call the method several times, all the specified sources are added to the test runner.
+You cannot call `src` multiple times for the same test runner. To provide multiple test files and/or templates, pass an array.
```

Comment:
Should we write "You cannot call" for each method? I think it's enough to add this info in the migration guide.